apps/explorer/src/ui/DescriptionList.tsx

```diff
@@ -11,7 +11,7 @@ export interface DescriptionItemProps {
 export function DescriptionItem({ title, children }: DescriptionItemProps) {
     return (
         <div className="flex flex-col gap-2 md:flex-row md:items-center md:gap-10">
-            <dt className="w-full flex-shrink-0 text-pBody font-medium text-steel-darker md:w-50">
+            <dt className="w-full flex-shrink-0 text-pBody font-medium text-steel-darker md:w-40">
```

Comment:
Just confirming that this change to description lists is intentional? cc @mystie711